### DISCUSSION TITLE
Remove the Outline of a Checkbox possible?

### DISCUSSION THREAD

**Discussion by tonka3000** [bdlukaa/fluent_ui#1109] [General]
Hi,

is there a way to remove the Outline (or restyle) of the `Checkbox` when it has the focus?

![image](https://github.com/user-attachments/assets/7f3f0caf-c518-402d-b685-06116fdef589)

Thanks in advance

**Comment by bdlukaa** [comment]
Hello!

You can wrap the checkbox in a `FocusTheme` and provide a custom border. You can also change the `focusThemeData` property in your `FluentTheme`.

**Reply by tonka3000** [reply]
Awesome. Thanks for your reply and the awesome library.

Just add the following example if somebody needed it too.

```dart
theme: FluentThemeData(
  focusTheme: const FocusThemeData(
    primaryBorder: BorderSide(style: BorderStyle.none),
    secondaryBorder: BorderSide(style: BorderStyle.none),
  ),
),
```

![image](https://github.com/user-attachments/assets/694b1a65-0a2a-411c-825c-1a73e41d1eb6)
